index.html

```diff
@@ -159,6 +159,26 @@ <h3 id="configuration">
         Which distribution of Debian the package should be built in. By
         default, this is automatically detected from the branch name.
       </dd>
+      <dt>
+        TRAVIS_DEBIAN_DERIVATIVE
+      </dt>
+      <dd>
+        Which Debian derivative to use, like "ubuntu". This will use the corresponding
+        <tt>TRAVIS_DEBIAN_DERIVATIVE:DISTRIBUTION</tt> Docker container as a base.
+        Some derivatives are automatically derived from <tt>TRAVIS_DEBIAN_DISTRIBUTION</tt>, and
+        defaults values for <tt>TRAVIS_DEBIAN_MIRROR</tt> and <tt>TRAVIS_DEBIAN_COMPONENTS</tt>
+        are then set. You can still override them as needed.
+        Note that not all features like backports, autopkgtests or security updates may be working,
+        depending on your derivative.
+      </dd>
+      <dt>
+        TRAVIS_DEBIAN_COMPONENTS
```

Comment:
But this is in another PR..?

Reply:
It is, however, as I'm setting TRAVIS_DEBIAN_COMPONENTS below based on the distribution, this PR is building upon the other one. Once the other one is merged in master, I'll just rebase on master (/me dreams of a feature like in launchpad where you can have dependencies between PR and only see the relevant diff). Making sense?